lessons/projects/5.mdx

```diff
@@ -0,0 +1,1379 @@
+                    <h3>{latestNFTMinted.name}</h3>
+                    <Image src={latestNFTMinted.image} height='200px' width='200px' />
+                  </div>
+                  <div style={modalFooter}>
+                    <button style={modalButton}>
+                      <a href={`https://testnets.opensea.io/assets/goerli/${CONTRACT_ADDRESS}/${tokenData}`} target="_blank">
+                        View on OpenSea
+                      </a>
+                    </button>
+                    <button style={modalButton}>
+                      <a href={`https://goerli.etherscan.io/tx/${mintData.hash}`} target="_blank">
+                        View on Etherscan
+                      </a>
+                    </button>
+                    <button onClick={() => setModalShow(false)} style={modalButton}>Close</button>
+                  </div>
+                </div>
+              }
+            </div>
+          )}
+        </main>
+        :
+        <main className={styles.main}>
+          <div>Please connect your wallet.</div>
+        </main>
+      }
+    </div>
+  )
+}
+
+const header = {
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'space-between'
+}
+
+const NFTFlex = {
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'space-evenly',
+  gap: '50px',
+}
+
+const NFTCard = {
+  display: 'flex',
+  flexDirection: 'column',
+  border: '2px solid white',
+  borderRadius: '10px',
+  padding: '20px',
+  alignItems: 'center',
+  gap: '10px',
+  fontWeight: 'bold',
+}
+
+const NFTMint = {
+  fontWeight: '700',
+  padding: '5px 20px',
+  border: '2px solid white',
+  color: 'white',
+  backgroundColor: 'black',
+  borderRadius: '5px',
+  cursor: 'pointer',
+}
+
+const modal = {
+  position: 'fixed',
+  left: '0',
+  top: '0',
+  right: '0',
+  bottom: '0',
+  backgroundColor: 'rgba(0, 0, 0, 0.8)',
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  textAlign: 'center',
+  zIndex: '1',
+}
+
+const modalContent = {
+  backgroundColor: '#fff',
+  padding: '10px 30px',
+  borderRadius: '16px',
+  color: '#000'
+}
+
+const modalBody = {
+  padding: '20px',
+  borderTop: '1px solid #eee',
+  borderBottom: '1px solid #eee',
+}
+
+const modalFooter = {
+  display: 'flex',
+  flexDirection: 'column',
+  gap: '10px',
+  padding: '20px',
+  justifyContent: 'space-evenly'
+}
+
+const modalButton = {
+  padding: '10px 20px',
+  backgroundColor: '#fff',
+  color: '#666',
+  border: '0',
+  borderRadius: '10px',
+  fontSize: '18px',
+  fontWeight: '700',
+  boxShadow: '0 0.2em 0.4em 0 rgba(0, 0, 0, 0.2), 0 0.3em 1em 0 rgba(0, 0, 0, 0.19)',
+  cursor: 'pointer',
+}
+```
+</ ContentSideDrawer>
+<br/>
+<br/>
+
+---
+
+## Fin.
+
+**Hoorah! We've done it! Pat yourselves on the back and get some ice cream.**
+
+Right from learning the basics of smart contract development using 
+solidity to creating a full stack decentralised NFT Minting Application.
+
+**Stay tuned for more!**
```

Comment:
Beautiful finish to a great and extremely well crafted lesson! Well done @ropats16 !